frontend/src/store/modules/dashboard/actions.js

```diff
@@ -114,9 +80,18 @@ export default {
 
     [RELOAD_ACTIVITY_DATA_ITEMS]: (context) => {
         const data = context.state.activityData.items.filter(item =>
-            moment().diff(moment(item.timeNotification), 'minutes') < 5
+            moment().diff(moment(item.date), 'minutes') < 5
         );
         context.commit(SET_ACTIVITY_DATA_ITEMS, data);
     },
 
+    [REFRESH_ACTIVITY_DATA_ITEMS]: (context, data) => {
+        const items =  context.state.activityData.items;
+        const index = _.indexOf(items, _.find(items, {url: data.url, visitor: data.visitor}));
+        if(index >= 0) {
+            items.splice(index, 1);
```

Comment:
I am not sure you can mutate state inside the action. You should do it inside the mutator I suppose. @TatsuUkraine am I right?

Reply:
Yep

Reply:
I would also replace index searching approach with findIndex method

Reply:
or event replace it with filter and spread operator